docs/source/markdown/options/volume.md

```diff
@@ -81,7 +81,9 @@ objects on the shared volumes. The **z** option tells Podman that two or more
 content with a shared content label. Shared volume labels allow all containers
 to read/write content. The **Z** option tells Podman to label the content with
 a private unshared label Only the current <<container|pod>> can use a private
-volume. Relabeling walks the file system under the volume and changes the label
+volume. Note: all containers within a `pod` share the same SELinux label. This
+means all containers within said pod can read/write volumes create with the
+`:Z`. Relabeling walks the file system under the volume and changes the label
```

Comment:
"without the `:Z` option."

Reply:
Does not matter, since they all run with the same label.  Only one relabel will be done, and if all containers run with the same MCS Label, they can use the volume (If the volume was mounted into the container.